mc-crusher.c

```diff
@@ -799,70 +680,121 @@ static void parse_config_line(char *line) {
         case PORT:
             template.port_num = atoi(value);
             break;
+        case THREAD:
```

Comment:
It bundles all of the connections for that line into its own thread. So if you say "conns=50,thread=true" you get 50 connections running that test in its own thread.

so, yep.